app/views/idv/in_person/state_id.html.erb

```diff
@@ -128,19 +154,21 @@
             required: true,
           ) %>
   <% end %>
-  <div class="margin-bottom-4">
-    <%= render ValidatedFieldComponent.new(
-          name: :state_id_jurisdiction,
-          collection: us_states_territories,
-          form: f,
-          hint: t('in_person_proofing.form.state_id.state_id_jurisdiction_hint'),
-          label: t('in_person_proofing.form.state_id.state_id_jurisdiction'),
-          label_html: { class: 'usa-label' },
-          prompt: t('in_person_proofing.form.state_id.state_id_jurisdiction_prompt'),
-          required: true,
-          selected: pii[:state_id_jurisdiction],
-        ) %>
-  </div>
+  <% unless capture_secondary_id_enabled %>
+    <div class="margin-bottom-4">
+      <%= render ValidatedFieldComponent.new(
+            name: :state_id_jurisdiction,
```

Comment:
I take it that this field went out live ahead of DAV?

Reply:
The field `:state_id_jurisdiction` already existed in the codebase before we started work on DAV.